mlir/utils/performance/perfRunner.py

```diff
@@ -33,7 +33,7 @@
 DATA_TYPES_GEMM = ['f32', 'f16', 'i8']
 
 # Compiled regexp object used for extracting elapsed time from MIOpenDriver's output
-ELAPSED_TIME_RE = re.compile(r"Elapsed: (.*)ms")
+ELAPSED_TIME_RE = re.compile(r"Elapsed: ([0-9\.]*) ms")
```

Comment:
I briefly used "--wall 2" which printed a line like "Elapsed: 9 ms  aux time 3 ms" and the original regexp matched "9 ms  aux time 3".  But it also printed two lines with elapsed time so I stopped using it.